### PR TITLE
Patch: more table styles

### PR DESCRIPTION
**Pull request by hollg**
Updates table styles after some changes to designs and some resolved miscommunications.

Screenshot 2021-11-11 at 08 18 18

### CODE REVIEW

**Comment by thomasdigby** [comment]
Remember to commit using the semantic message style, none of these commits will be picked up by semantic release and it won't publish a new version. We can still merge and use @kerryambt's update to publish your changes

File: src/components/table/Table.tsx [hunk 1]
```diff
@@ -21,22 +21,25 @@ type TableSubComponents = {
...
   variants: {
     size: {
       md: {
-        [`${TableCell}`]: {
+        [`${TableCell}, ${TableHeaderCell}`]: {
           height: '$4'
         },
         [`${TableHeaderCell}`]: {
           height: '$4'
         }
```
Comment (thomasdigby) [inline src/components/table/Table.tsx:36]: We can remove this extra selector/style here

**Review by thomasdigby — APPROVED** [review]

**Comment by hollg** [comment]
> Remember to commit using the semantic message style, none of these commits will be picked up by semantic release and it won't publish a new version. We can still merge and use @kerryambt's update to publish your changes

Really? I started a commit with `patch` but the precommit hook stopped me because `patch` isn't a valid commit type. It gave a list of valid types which included `style` so that's what I used. Is that precommit hook not aligned with what the semantic-release package?

**Comment by hollg** [comment]
@thomasdigby I've added an empty `fix` commit so I think it should get picked up now

**Review by Mhoog — APPROVED** [review]

**Comment by thomasdigby** [comment]
> > Remember to commit using the semantic message style, none of these commits will be picked up by semantic release and it won't publish a new version. We can still merge and use @kerryambt's update to publish your changes
> 
> 
> 
> Really? I started a commit with `patch` but the precommit hook stopped me because `patch` isn't a valid commit type. It gave a list of valid types which included `style` so that's what I used. Is that precommit hook not aligned with what the semantic-release package?

Ah apologies, might have jumped the gun there. I would double check that it's aligned with semantic release so we're sure that it'll trigger a release correctly, I'm not sure what the overlap is.

**Comment by andoulla** [comment]
> @thomasdigby I've added an empty `fix` commit so I think it should get picked up now

We have a commit linter that shouldn't allow commits that don't follow the semantic format. I remember setting it up and it used to prevent most mistakes.

**Pull request merged**